### PR TITLE
Adding auto assignment to pr

### PR DESCRIPTION
**Pull request by VALERA771**
If this PR will be merged, I’ll do same thing for dev branch

### CODE REVIEW

File: .github/workflows/auto-assigning.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    types: [ opened, reopened ]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-author:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@v2.1.1
```
Comment (ALEXWARELLC) [inline .github/workflows/auto-assigning.yml:14]: With the size of this GitHub Action, this should actually be moved into the labeller and rename the labeller to `pull_request_opened.yml`.

This gives the workflow file a meaningful name and compresses what is the same action into 1, easy to identify file.

File: .github/workflows/auto-assigning.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: Auto Author Assign
+
+on:
+  pull_request_target:
+    types: [ opened, reopened ]
```
Comment (ALEXWARELLC) [inline .github/workflows/auto-assigning.yml:5]: No need to add a reopening trigger due to the author already being added when the Pull Request is initially created.

**Review by ALEXWARELLC — CHANGES_REQUESTED** [review]
Please read through.

Thanks.

**Comment by ALEXWARELLC** [comment]
Also, please make sure this workflow has been tested and works.

**Comment by VALERA771** [comment]
Tested (bruh where label)

**Review by ALEXWARELLC — APPROVED** [review]
👍

**Review by louis1706 — APPROVED** [review]

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**